mm2src/mm2_main/tests/mm2_tests/mm2_tests_inner.rs

```diff
@@ -7236,6 +7238,34 @@ fn test_gui_storage_coins_functionality() {
     assert_eq!(actual.result, expected);
 }
 
+#[test]
+#[cfg(not(target_arch = "wasm32"))]
+fn test_btc_block_header_sync_with_optimzation() {
```

Comment:
This tests that block headers sync completes successfully, which is good. But can you think of a way to test that the excessive block headers get deleted before `max_stored_block_headers` is surpassed?

Reply:
this can be checked in `[test_block_header_utxo_loop](https://github.com/KomodoPlatform/atomicDEX-API/blob/89f7207518e4eb7a8194e388ddfcecff7f4dbd0f/mm2src/coins/utxo/utxo_tests.rs#L4279)`